CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 
 Unreleased
 ==========
-
+* feat: MenuContent and MenuItem links open in sideframe
```

Comment:
It's important when looking at a changelog you can digest the extent of changes. If I am looking into the changelog because my icons / buttons broke this entry would tell me it was potentially this change.  

```suggestion
* feat: MenuContent and MenuItem links open in sideframe (refactored icons)
```